runtime/bindings/python/src/openvino/ie_api.py

```diff
@@ -45,6 +46,11 @@ def infer(request: InferRequest, inputs: dict = None) -> np.ndarray:
     res = request._infer(inputs=normalize_inputs(inputs if inputs is not None else {}))
     return np.asarray([copy.deepcopy(tensor.data) for tensor in res])
 
+
+def infer_new_request(exec_net: ExecutableNetwork, inputs: dict = None) -> np.ndarray:
+    res = exec_net._infer_new_request(inputs=normalize_inputs(inputs if inputs is not None else {}))
+    return np.asarray([copy.deepcopy(tensor.data) for tensor in res])
```

Comment:
Remove `np.asarray`, I recently found out that this approach broke some of the cases where outputs can differ in sizes.

Reply:
done